pyproject.toml

```diff
@@ -50,6 +50,7 @@ python-box = "^7.1.1"
 pypdf = "^3.17.1"
 pyyaml = "^6.0.1"
 chromadb = "^0.4.18"
+pyautogen = {version = "^0.2.2", optional = true}
 
 [tool.poetry.extras]
 local = ["torch", "huggingface-hub", "transformers"]
```

Comment:
can you also add an extras section here called "autogen" that uses "pyautogen"

Reply:
Sure,